libs/testing/src/notification-template.service.ts

```diff
@@ -42,6 +42,14 @@ export class NotificationTemplateService {
             url: '/cypress/test-shell/example/test?test-param=true',
           },
         },
+        variables: [
+          {
+            defaultValue: '',
+            name: 'firstName',
+            required: false,
+            type: 'String',
+          },
+        ],
```

Comment:
Added the missing variables array, after I've fixed all the variable management issues the tests were not passing because of this causing the form to be dirty.